deepfence_server/handler/cloud_node.go

```diff
@@ -324,3 +325,42 @@ func (h *Handler) CachePostureProviders(ctx context.Context) error {
 	}
 	return nil
 }
+
+func (h *Handler) DeleteCloudAccountHandler(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
+	var req model.CloudAccountDeleteReq
+	err := httpext.DecodeJSON(r, httpext.NoQueryParams, MaxPostRequestSize, &req)
+	if err != nil {
+		log.Error().Msgf("%v", err)
+		h.respondError(&BadDecoding{err}, w)
+		return
+	}
+
+	log.Info().Msgf("delete cloud accounts request: %v", req)
+
+	if len(req.NodeIDs) > 0 {
+		worker, err := directory.Worker(r.Context())
+		if err != nil {
+			log.Error().Msgf("%v", err)
+			h.respondError(&InternalServerError{err}, w)
+			return
+		}
+
+		data, err := json.Marshal(req)
+		if err != nil {
+			log.Error().Err(err).Msg("failed to marshal cloud account delete request")
+			h.respondError(&InternalServerError{err}, w)
+			return
+		}
+
+		if err := worker.Enqueue(utils.DeleteCloudAccounts, data, utils.CritialTaskOpts()...); err != nil {
+			log.Error().Err(err).Msg("failed enqueue task ddelete cloud accounts")
+			h.respondError(&InternalServerError{err}, w)
+			return
+		}
+	}
+
+	h.AuditUserActivity(r, EventComplianceScan, ActionDelete, req, true)
```

Comment:
I think auditlogs should be created even if the task fails. 
we need a better framework at router level for auditlog, instrumenting all the methods with `h.AuditUserActivity` is overkill.(not scope of this PR)